master/reference/calicoctl/resources/ippool.md

```diff
@@ -17,7 +17,8 @@ metadata:
   cidr: 10.1.0.0/16
 spec:
   ipip:
-    enabled: false
+    enabled: true
```

Comment:
that seems better. Just curious, do we validate if ipip is disabled and mode set to something?